test.js

```diff
@@ -1,10 +1,17 @@
 'use strict';
-const app = require('app');
-const BrowserWindow = require('browser-window');
+const electron = require('electron');
+const app = electron.app;
+const BrowserWindow = electron.BrowserWindow;
 
 require('.')({showDevTools: true});
 
+function load(url) {
+	const win = new BrowserWindow({show: true});
+	win.loadURL(url);
+	win.setMenu(null);
```

Comment:
Here I disabled builtin menu to ensure we are testing electron debug shortcuts and not menu ones